WcaOnRails/app/views/shared/_associated_events_picker.html.erb

```diff
@@ -32,16 +34,21 @@ Note: form_builder.object is the object having associated events.
   <% end %>
   <div id="<%= events_association_name %>" class="associated-events">
     <%= form_builder.simple_fields_for events_association_name, all_events do |f| %>
-      <span class="event-checkbox <%= "disabled" if disabled %>">
-        <% event = f.object.event %>
+      <% event = f.object.event %>
+      <% e_disabled = disabled_events.include?(event) %>
+      <span class="event-checkbox <%= "disabled" if disabled || e_disabled %>">
         <% if using_competition_events %>
           <%= f.hidden_field :competition_event_id %>
         <% else %>
           <%= f.hidden_field :event_id %>
         <% end %>
         <%= label_tag "#{events_association_name}_#{event.id}" do %>
-          <%= f.check_box "_destroy", { checked: selected_events.include?(event), id: "#{events_association_name}_#{event.id}", disabled: disabled }, "0", "1" %>
-          <%= cubing_icon(event.id, data: { toggle: "tooltip", placement: "top" }, title: event.name) %>
+          <%= f.check_box "_destroy", { checked: selected_events.include?(event) && !e_disabled, id: "#{events_association_name}_#{event.id}", disabled: disabled || e_disabled }, "0", "1" %>
+          <% tooltip = e_disabled ? disabled_tooltip : event.name %>
+          <%= cubing_icon(event.id, data: { toggle: "tooltip", placement: "top" }, title: tooltip) %>
```

Comment:
Can we have the icon appear in some gentle tone of red (along the lines of `#FF7F7F`) when you're not qualified? Just to distinguish between "It's not selected because you haven't clicked it yet" and "It's not selected because you _cannot_ click it"

Reply:
Fine by me!